### PR TITLE
evmos deployment script

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contracts/adapters/TokenRegistry.sol [hunk 1]
```diff
@@ -10,11 +10,13 @@ import "../../interfaces/IBZx.sol";
 
 contract TokenRegistry {
 
-    address public constant bZxContract = 0xD8Ee69652E4e4838f2531732a46d1f7F584F0b7f; // mainnet
+    //address public constant bZxContract = 0xD8Ee69652E4e4838f2531732a46d1f7F584F0b7f; // mainnet
```
Comment (RomanHiden) [inline contracts/adapters/TokenRegistry.sol:13]: can you please add evmos addys but leave mainnet as default one ?

**Comment by Drypto13** [comment]
make sure to rebase the PR to development branch.

File: contracts/adapters/TokenRegistry.sol (hunk 1 above)
Comment (RomanHiden) [reply contracts/adapters/TokenRegistry.sol:13]: @eugenesh1

**Comment by RomanHiden** [comment]
you deployed from development so you should merge there too

File: contracts/feeds/PriceFeeds_EVMOS.sol [hunk 2]
```diff
@@ -0,0 +1,391 @@
+/**
```
Comment (Drypto13) [inline contracts/feeds/PriceFeeds_EVMOS.sol:1]: This file can be removed as it is not needed. Deploy the normal PriceFeeds.sol contract instead

Comment (RomanHiden) [reply contracts/feeds/PriceFeeds_EVMOS.sol:1]: @eugenesh1 consider this

**Pull request closed without merging**